spaces/domains.json

```
@@ -977,4 +977,5 @@
   "vote.evelon.app": "evelonapp.eth",
   "vote.zkswap.finance": "zkswapfinance.eth",
   "vote.gamedao.co": "gamedao.eth"
+  "vote.reformdao.com": "rfrmdao.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "vote.gamedao.co": "gamedao.eth",
  "vote.reformdao.com": "rfrmdao.eth"
```